### DISCUSSION TITLE
No auto clear or documentation for clearing logs ?

### DISCUSSION THREAD

**Discussion by whitespacecode** [megahertz/electron-log#398] [Ideas]
Is there a built-in mechanism to clear old logs? It seems like a useful feature that could be added.

I'm thinking of a setting that checks the size of the log file or its creation date, allowing users to set a specific time period for log retention.

If there is not such a functionality available, could there be documentation on how to implement a 'clear log' into the application?

**Comment by megahertz** [comment]
You can set a custom `log.transports.file.archiveFn`.

**Reply by whitespacecode** [reply]
I see there already is a `archiveLogFn`. Good to know. Maybe the readme could be updated

**Reply by megahertz** [reply]
It's already described in docs
